code/datums/components/plumbing/IV_drip.dm

```diff
@@ -1,21 +1,31 @@
-///Component that feeds whatever it's supplied with to whoever is buckled to it. Can be safely added to whatever like the others, like chairs or something
+///Component for IVs that tracks the current person being IV'd. Input received through plumbing is instead routed to the whoever is attached
 /datum/component/plumbing/iv_drip
 	demand_connects = SOUTH
 	supply_connects = NORTH
 
 	methods = INJECT
 
-/datum/component/plumbing/IV_drip/Initialize()
+/datum/component/plumbing/iv_drip/RegisterWithParent()
 	. = ..()
 
 	RegisterSignal(parent, list(COMSIG_IV_ATTACH), .proc/update_attached)
 	RegisterSignal(parent, list(COMSIG_IV_DETACH), .proc/clear_attached)
 
 	recipient_reagents_holder = null
```

Comment:
This should probably still be in Initialize, RegisterWithParent is *just* for registering signals.

Reply:
You’re right, I have no idea what kind of brainfart allowed me to do this